packages/build/tests/plugins/tests.js

```diff
@@ -332,6 +332,50 @@ const waitsForPostProcessing = function (request) {
   return request.action === 'deploySiteAndAwaitLive'
 }
 
+test('DPC plugin is not used on other contexts', async (t) => {
+  await runFixture(t, 'empty', { flags: { featureFlags: 'dpc' } })
+})
+
+test('DPC plugin is only used if the feature flag is enabled', async (t) => {
+  await runFixture(t, 'empty', { env: { CONTEXT: 'deploy-preview' } })
+})
+
+test('DPC plugin shows a URL as a plugin status', async (t) => {
+  await runFixture(t, 'empty', { flags: { featureFlags: 'dpc' }, env: { CONTEXT: 'deploy-preview' } })
+})
+
+test('DPC plugin should modify netlify.toml headers', async (t) => {
+  await runFixture(t, 'dpc_headers_toml', {
+    flags: { featureFlags: 'dpc' },
+    env: { CONTEXT: 'deploy-preview' },
+    copyRoot: { git: false },
```

Comment:
`copyRoot` creates a copy of the fixture directory in `/tmp`, then uses it instead. This is required since the DPC plugin modifies files.

Reply:
Oh this is neat!